src/commands/run-tests.yml

```diff
@@ -1,105 +1,113 @@
 description: >
-  Run all tests in a MATLAB project and generate test artifacts. MATLAB includes any files in your
-  project that have a Test label. If your pipeline does not leverage a MATLAB project, then MATLAB
-  includes all tests in the the root of your repository including its subfolders.
+  Run MATLAB and Simulink tests and generate artifacts. By default, MATLAB includes any files in
```

Comment:
Do you like to replace "MATLAB includes" (x2) in this paragraph with "the command includes"?